keyboards/handwired/ferris/keymaps/json2crab.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
```

Comment:
Is this necessary?  

I believe that we do have `qmk json2c` that will handle this, as well.

Reply:
I may have missed something, but I thought json2c only converted from json to c code. What this script is doing is going from unformatted json to formatted json that is more human readable (formatting by half rows of 5 keys and lining up the columns).

Compared to checking out unformatted json from the configurator,
This allows:
* better git diffs (smaller lines)
* easier to read the json without loading it in the configurator
* possible to hand-edit the keymap for small changes without using the configurator

See `default/keymap.json` for an example of output from that script (input is a single line of json coming from the configurator)

If I'm missing a formatting option of json2c, please let me know :)

Reply:
Hi @drashna , any comment on the questions above?